### PR TITLE
your message

### PR DESCRIPTION
**Pull request by alexrz07**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -32,7 +33,13 @@ console.log('hello world')
  * Example Output: "Elizabeth Thompson"
  */
 
-// TODO: ADD CODE HERE
+const firstName = 'Alex';
+const lastName = 'Rodriguez';
+console.log(firstName + ' ' + lastName);
```
Comment (ethomp-dev) [inline index.js]: You could also use a template literal here!

Read more about [Template Literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)

File: index.js [hunk 2]
```diff
@@ -88,7 +118,14 @@ console.log('hello world')
  * Example Output: "I called an arrow function!"
  */
 
-// TODO: ADD CODE HERE
+myArrowFuction = () => {
```
Comment (ethomp-dev) [inline index.js]: When you define an arrow function, the variable name it's stored in needs a keyword:

```js
const myArrowFunction = () => {}
```

File: index.js [hunk 3]
```diff
@@ -102,7 +139,17 @@ console.log('hello world')
  * Example Output: "WHY AM I SCREAMING"
  */
 
-// TODO: ADD CODE HERE
+let str = "Ctd Rocks 🤘🏼!";
+
+stringToUpper = (str) => {
```
Comment (ethomp-dev) [inline index.js]: When you define an arrow function, the variable name it's stored in needs a keyword:

```js
const stringToUpper = () => {}
```

File: index.js [hunk 4]
```diff
@@ -118,5 +165,13 @@ console.log('hello world')
  *
  * Example Output: 2, 10, 20
  */
+multiplyByTwo = (number) => {
```
Comment (ethomp-dev) [inline index.js]: When you define an arrow function, the variable name it's stored in needs a keyword:

```js
const multiplyByTwo = () => {}
```

**Review by ethomp-dev — COMMENTED** [review]
@alexrz07 looks great! 🎉 

Just a few comments...

File: index.js (hunk 1 above)
Comment (alexrz07) [reply index.js]: Hi Elizabeth, Thank you so much for reviewing my code and for the comments, I just fixed them, can you please review it again if you don't mind. do i need to create a new pull request?

**Review by ethomp-dev — APPROVED** [review]
@alexrz07 great work! 🎉

**Pull request merged**